shared/chat/manage-channels/index.desktop.js

```diff
@@ -114,7 +114,7 @@ const ManageChannels = (props: Props) => {
           {props.teamname}
         </Text>
         {channelDisplay}
-        <ScrollView style={{alignSelf: 'flex-start', width: '100%', paddingBottom: globalMargins.xlarge}}>
+        <ScrollView style={{alignSelf: 'flex-start', marginBottom: globalMargins.xlarge, width: '100%'}}>
```

Comment:
So after playing around with this in the devtools a little more, the reason why this change works is because the `ScrollArea` component takes on the height of the content inside. This works perfectly fine unless there are a lot of channels – then the content's height is bigger than the available room for `ScrollArea`.

`marginBottom` essentially just removes the bottom 64 pixels. 

The `ButtonBar` has a property of `justify-content: flex-end`, so it will always be put at the bottom of the modal. With the margin, this creates the perfect illusion of the `ScrollArea` stopping right before the `ButtonBar`. Without the margin, `ScrollArea` continues until it is stopped by the bounding `Box` and we get the buttons hovering over the scroll area.

If we don't want to add margin to the bottom of the `ScrollArea`, we could also explicitly set the height of the `ScrollArea` to 320px, the height it is when it has a bottom margin of 64.

I don't think we can do anything with flexbox to change how this works as `ScrollArea` will just overflow with content.

So I think the only two solutions are:
1. Explicit bottom margin
2. Explicit height
but I could be missing something. Let me know what you want to do.